src/test/java/accounts/VerifyElementsOnAccountPageTest.java

```diff
@@ -0,0 +1,31 @@
+package accounts;
+
+import base.BaseTest;
+import data.UserData;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.AccountPage;
+import pages.HomePage;
+import pages.LogInPage;
+
+public class VerifyElementsOnAccountPageTest extends BaseTest {
+    public LogInPage logInPage;
+    public HomePage homePage;
+    public AccountPage accountPage;
+    SoftAssert softAssert = new SoftAssert();
+
+    @BeforeTest
+    public void login() {
+        logInPage = base.openPlayGroundPage();
+        homePage = logInPage.logInPlayGround(UserData.ADMIN_USER_EMAIL, UserData.ADMIN_USER_PASSWORD);
+    }
+
+    @Test(priority = 1, description = "Verify opening Account Page")
+    public void accountsPageIsOpenedTest() {
+        homePage.getHomePageNavBarText();
+        accountPage = homePage.openAccountPage();
+        softAssert.assertEquals(accountPage.getAccountsPageName(), "Accounts", "Accounts Page should have name 'Accounts'.");
```

Comment:
If you have only one assertion - use Assert instead of soft asset. Soft assert uses only if you have more than one assertion in test.

Reply:
@Evangard done